packages/cb-check/cb-check.0.1/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Json schema checker for current-bench"
+maintainer: ["Ambre Austen Suhamy <ambre@tarides.com>"]
+authors: ["Ambre Austen Suhamy <ambre@tarides.com>"]
+license: "Apache-2.0"
+homepage: "https://github.com/ocurrent/current-bench"
+bug-reports: "https://github.com/ocurrent/current-bench/issues"
+depends: [
+  "dune" {>= "3.7"}
+  "ocaml" {>= "4.13.0"}
+  "yojson"
+  "uri" {with-test}
+  "ppx_deriving_yojson" {with-test}
+  "timere-parse" {with-test}
+  "rresult" {with-test}
+  "prometheus" {with-test}
+  "logs" {with-test & >= "0.5.0"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
```

Comment:
```suggestion
```

if you decide to go with this, please remove the test dependencies added above

Reply:
Done!